pyproject.toml

```diff
@@ -26,6 +26,7 @@ dependencies = [
     "beartype",
     "tabulate",
     "filetype",
+    "pytest-xdist",
```

Comment:
This is a test dependency.

We can separate test dependencies into a distinct section of this file
```
[project.optional-dependencies]
test = ["pytest-xdist"]
```

Reply:
fixed